packages/ui/src/components/Button.svelte

```diff
@@ -24,14 +24,15 @@
   export let labelParams: Record<string, any> = {}
   export let kind: 'primary' | 'secondary' | 'no-border' | 'transparent' | 'link' | 'dangerous' = 'secondary'
   export let size: 'small' | 'medium' | 'large' | 'x-large' = 'medium'
+  export let shape: 'circle' | undefined
```

Comment:
```suggestion
  export let shape: 'circle' | undefined = undefined
```

To prevent warnings for usual use

Reply:
Updated